public/views/question.html

```diff
@@ -1,38 +1,39 @@
+            <div id="finding-players">Finding Players</div>
+            <div id="invite-friend"></div>
+            <div id="player-count-container">
+                <div id="player-count">{{game.players.length}} / 12 </div>
+                <div id="the-word-players"> Players </div>
+            </div>
+            <div id="loading-container">
+                <div id="loading-gif"><img ng-src="../img/loader.gif" /></div>
+            </div>
+            <div id="start-game-container" ng-click="checkPlayerLimit()" ng-show="(game.playerIndex === 0 || game.joinOverride)">
+                <div id='start-game-button'>
+                    Start Game<br>with {{game.players.length}} players
+                </div>
+            </div>
         </div>
-      </div>
-    </div>
-    <!-- Game Over Stuff -->
-    <div id="game-end-info">
-      <div id="people-left-game-top" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
-        <div class="game-end-headline">What the!? The game ended because too many people left! </div>
-        Our sincere apologies.
-      </div>
-      <div ng-show="game.state === 'game ended' && game.gameWinner === game.playerIndex">
-        <div class="game-end-headline">Congratulations! You won the game, {{game.players[game.playerIndex].username | upperFirstLetter}}!</div>We hope you feel special. If you'd like to share that special feeling with the world, consider donating to charity. Or just horde it all for youself.
-      </div>
-      <div ng-show="game.state === 'game ended' && game.gameWinner !== game.playerIndex">
-        <div class="game-end-headline">Aw, shucks! {{game.players[game.gameWinner].username | upperFirstLetter }} won the game!</div>
-        That makes you a loser, unfortunately.
-      </div>
-    </div>
-  </div>
-</div>
 
+        <!-- Game Over Stuff -->
+        <div id="game-end-info">
+            <div id="people-left-game-top" ng-show="game.state ==='game dissolved' && game.gameWinner === -1">
+                <div class="game-end-headline">What the!? The game ended because too many people left! </div>
+                Our sincere apologies.
+            </div>
+            <div ng-show="game.state === 'game ended' && game.gameWinner === game.playerIndex">
+                <div class="game-end-headline">Congratulations! You won the game, {{game.players[game.playerIndex].username | upperFirstLetter}}!</div>We hope you feel special. If you'd like to share that special feeling with the world, consider donating to charity. Or just horde it
+                all for youself.
+            </div>
+            <div ng-show="game.state === 'game ended' && game.gameWinner !== game.playerIndex">
+                <div class="game-end-headline">Aw, shucks! {{game.players[game.gameWinner].username | upperFirstLetter }} won the game!</div>
+                That makes you a loser, unfortunately.
+            </div>
+        </div>
+    </div>
```

Comment:
Always add a newline at end of a file. It's easier to set your editor to add it automatically. If using visual studio code, add the following to your settings.

```json
{
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true
}